resotocore/requirements-dev.txt

```diff
@@ -8,3 +8,6 @@ coverage==6.3.2
 setuptools==60.10.0
 pylint==2.12.2
 black==22.1.0
+# Explicit dependency can be removed when this has been resolved.
+# https://github.com/dask/distributed/issues/6013
+click <= 8.0.4
```

Comment:
Isn't specifying click in resotolib test dependency enough?

Reply:
I just bumped black==22.3.0 which fixes the click dependency. No need to add click to our own dependencies.